src/context.js

```diff
@@ -107,6 +161,31 @@ function EvolvContext(store) {
     return true;
   };
 
+  /**
+   * Persist context key to sessionStorage.
+   *
+   * @param {String} key The key to persist.
+   * @param {boolean} [sessionOnly] If true, the value will only be persisted to sessionStorage.
+   *   Default: true
+   */
+  this.persist = function(key, sessionOnly) {
+    ensureInitialized();
+
+    if (window === undefined || !window.localStorage) {
+      console.log('Evolv: Unable to persist context key. LocalStorage is not available.');
+      return;
```

Comment:
It might be worth returning a boolean to indicate if the value was stored successfully or not.
```suggestion
      console.warn('Evolv: Unable to persist context key. LocalStorage is not available.');
      return false;
```